### PR TITLE
add basic server

### PR DESCRIPTION
**Pull request by briarsweetbriar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,13 +2,17 @@
   "name": "@counterfactual/truffle-box",
   "version": "0.0.0",
   "description": "",
-  "main": "src/app.js",
+  "main": "truffle.js",
```
Comment (ebryn) [inline package.json:5]: why was this changed?

Comment (briarsweetbriar) [reply package.json:5]: To match how it's done [here](https://github.com/truffle-box/pet-shop-box/blob/master/package.json#L5).
